src/java.base/share/classes/java/util/concurrent/Executors.java

```diff
@@ -422,7 +422,7 @@ public static <T> Callable<T> callable(Runnable task, T result) {
      * @return a callable object
      * @throws NullPointerException if task null
      */
-    public static Callable<Object> callable(Runnable task) {
+    public static Callable<@Nullable Object> callable(Runnable task) {
```

Comment:
Isn't an import missing? Either I'm missing something or something about the CI is broken, because it is green for this PR.

Reply:
Oh, CI isn't actually doing anything meaningful, which isn't good. I'll look into fixing that.

Reply:
Sorry about that. I've added the import. Thanks for looking into the CI.

Reply:
Added https://github.com/eisop/checker-framework/issues/231